pkg/util/logutil/log.go

```diff
@@ -445,3 +445,22 @@ func SampleLoggerFactory(tick time.Duration, first int, fields ...zap.Field) fun
 		return logger
 	}
 }
+
+// SampleErrVerboseLoggerFactory returns a factory function that creates a sample logger with error verbose logging.
+// It works similarly to SampleLoggerFactory but ensures that error details are always logged,
+// regardless of the logging configuration.
+func SampleErrVerboseLoggerFactory(tick time.Duration, first int, fields ...zap.Field) func() *zap.Logger {
+	var (
+		once   sync.Once
+		logger *zap.Logger
+	)
+	return func() *zap.Logger {
+		once.Do(func() {
+			sampleCore := zap.WrapCore(func(core zapcore.Core) zapcore.Core {
```

Comment:
What's mean of this sample log with duration ?

Reply:
To ensure we don’t print too many identical logs in a short period.